org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Replay/ImportPolygon.java

```diff
@@ -169,7 +169,7 @@ else if (hasStartEndDates(line))
 		{
 		  // meet the label
 		  final String sts = st.nextToken();
-		  if (!isNameLabel(sts) && Character.isDigit(sts.charAt(0)))

```

Comment:
This may not have an immediate benefit to REP file import, but I really like the fact that you've identified and implemented the optimisation.